test/chat_api/reporting_test.exs

```diff
@@ -91,6 +91,24 @@ defmodule ChatApi.ReportingTest do
                Reporting.count_conversations_by_date(account.id)
     end
 
+    test "average_first_replied_time gets the average seconds it takes to respond", %{
```

Comment:
would be good to also test cases with more variability in reply time, as well as cases where some conversations have `first_replied_at: nil`